tests/transformers/test_feed_forward.py

```
@@ -0,0 +1,19 @@
+import torch
+
+from rtw_models.transformer.feed_forward import FeedForward
+
+
+class TestFeedForward:
```

Ruff (0.8.2)

Missing docstring in public class

(D101)